app/scripts/views/complete_reset_password.js

```diff
@@ -178,6 +180,13 @@ function (Cocktail, BaseView, FormView, Template, FloatingPlaceholderMixin,
         });
       })
       .fail(self.displayError.bind(self));
+    },
+
+    _clearSameBrowserVerificationInfo: function () {
```

Comment:
Maybe move this function into a mixin to be shared. Expose `urlVerificationInfo` as a public variable, or create a getter function.